.github/workflows/integration-tests.yml

```diff
@@ -123,3 +123,105 @@ jobs:
           flags: ${{ steps.test-coverage-flags.outputs.os }},${{ steps.test-coverage-flags.outputs.node }}
           token: ${{ secrets.CODECOV_TOKEN }}
         if: '${{ !steps.release-check.outputs.IS_RELEASE }}'
+  integration-win-node-23:
```

Comment:
What is the long-term plan for this action? I understand we're looking to test this specific version today, but what happens after this goes to LTS and beyond? Can this be removed? Just trying to make sure we're not left with this extra step forever without people understanding why. Perhaps adding a comment here would help?